argo-cd-apps/overlays/environment/delete-applications.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: argoproj.io/v1alpha1
```

Comment:
we do you delete the monitoring components? do you plan to add them in a following pr?

Reply:
We're planning to set them up later ([RHTAPWATCH-915](https://issues.redhat.com/browse/RHTAPWATCH-915)).